controllers/designate_controller.go

```diff
@@ -1096,6 +1097,48 @@ func (r *DesignateReconciler) generateServiceConfigMaps(
 	}
 	templateParameters["ServiceUser"] = instance.Spec.ServiceUser
 
+	transportURLSecret, _, err := oko_secret.GetSecret(ctx, h, instance.Status.TransportURLSecret, instance.Namespace)
+	if err != nil {
+		if k8s_errors.IsNotFound(err) {
+			Log.Info(fmt.Sprintf("TransportURL secret %s not found", instance.Status.TransportURLSecret))
+			instance.Status.Conditions.Set(condition.FalseCondition(
+				condition.InputReadyCondition,
+				condition.RequestedReason,
+				condition.SeverityInfo,
+				condition.InputReadyWaitingMessage))
+			return nil
+		}
+		instance.Status.Conditions.Set(condition.FalseCondition(
+			condition.InputReadyCondition,
+			condition.ErrorReason,
+			condition.SeverityWarning,
+			condition.InputReadyErrorMessage,
+			err.Error()))
+		return err
+	}
+	templateParameters["TransportURL"] = transportURLSecret.Data["transport_url"]
```

Comment:
AFAIK a Data of a Secret is an array of byte, so it means that TransportURL is not a string.
Can you check the generated config file and confirm that the transport_url value is well formed?

Reply:
Good catch, I changed it.